libs/ui-watt/src/lib/foundations/color/storybook/color.component.scss

```diff
@@ -0,0 +1,37 @@
+@use 'sass:map';
+@use './../colors.import' as colors;
+
+:host {
+    background: colors.getColor('grey-300');
+    display: block;
+    width: 100%;
+    padding: 1rem;
+    margin: -1rem;
+}
+
+mat-card {
+    margin-bottom: 8px;
+}
+
+.color-grid {
+    display: flex;
+    flex-wrap: wrap;
+    max-width: 688px;
+}
+
+figure {
+    margin: 10px;
+}
+
+figcaption {
+    text-transform: uppercase;
+}
+
+.color-sample {
+    height: 150px;
+    width: 150px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border: 1px solid #{colors.getColor('grey-300')};
```

Comment:
Suggestion: There's border only on the "white" color swatch in Figma. Consider aligning for consistency.

Reply:
This is done to avoid having to look for if the color is white as the colors are printed dynamically, which adds unnecessary logic and really is fragile as this properly only applies when the background is white so the real logic should be, that it should have a border if the color and background is the same or even the contrast aren't good enough. Alternative manually add the color-samples manually but really not better either, at least in my opinion. 
I don't think @stephaniegitha minds, it's barely visible and takes some time to fix which really doesn't give us any value besides matching figma perfectly.

Reply:
It's not only the value that we get but also the time that we use to create something. What is the point of creating something in one place and do it differently somewhere else? If Figma is the source of truth for the Design System, then we should align with it. And sometimes we might do something in Storybook which initiates changes in Figma. So it goes both ways. If we can align, let's do it.

Reply:
Totally agree, we should align